src/main/java/com/coursesPlatform/coursePortfolio/OfferService.java

```diff
@@ -19,13 +20,15 @@
     private final CategoryRepository categoryRepository;
     private final CourseRepository courseRepository;
     private final LectureRepository lectureRepository;
+    private final EmailFacade emailFacade;
 
-    OfferService(OfferRepository offerRepository, CustomerRepository customerRepository, CategoryRepository categoryRepository, CourseRepository courseRepository, LectureRepository lectureRepository) {
```

Comment:
I would put each parameter in separate line, this is long and becomes not readable.